modules/client-configuration/pages/autoinst-intro.adoc

```diff
@@ -1,28 +1,39 @@
-This is useful if you need to install a large number of clients.
-
-For most clients, you can use an autoinstallation file.
-The autoinstallation file contains information about the distribution to install on the clients.
-There are two main types of autoinstallation file:
-
-* For {sle} clients, use {ay}.
-* For {rhel} clients, use {kickstart}.
-
-When you have created an autoinstallation file, you can upload and manage it in the {productname} {webui}.
-The {webui} also provides tools to help you edit and maintain your autoinstallation files.
-For more information about autoinstallation files, see xref:client-configuration:autoinst-setup.adoc[].
-
-When you have created your autoinstallation files, you can create a profile to manage them.
-This makes it easier to store and manage multiple autoinstallation files for different types of clients.
-For more information about autoinstallation profiles, see xref:client-configuration:autoinst-profiles.adoc[].
-
-When you are ready to perform the autoinstallation, you need to make the {productname} Server aware of the client.
-This is called provisioning.
-You can provision a client by enabling bare metal provisioning.
-Using this method, when a client connects to an IP address within a given range, the autoinstallation begins.
-You can also use Cobbler and PXE booting to provision clients within your network.
-Alternatively, you can provision clients using the {productname} API.
-For more information about client provisioning, see xref:client-configuration:autoinst-provisioning.adoc[].
+A classic way to use {productname} is to register clients that are already running. You might have installed these machines manually just before registering them to {productname}, or they can have existed for years, even before {productname} has been introduced into your infrastructure.
+
+An alternative is to have {productname} help you install the operating systems on those machines, and register them to {productname} in one go. This installation will be mostly or totally automated, avoiding you to spend time answering installer questions, and allowing you to quickly install and register many systems.
+
+There are four ways to install an operating system from {productname}:
+
+. in-place on already registered {productname} clients;
+. via the network, using the PXE boot technology;
+. by preparing an installation CD-ROM and then going to the machine to boot on that medium;
+. as part of the {productname} {smr} solution.
+
+The in-place reinstallation assumes that:
+
+* a previous operating system had already been installed on the client;
+* the client had already been registered previously.
+
+For details about that method, please refer to xref:client-configuration:autoinst-reinstall.adoc[Reinstalling Registered Systems].
+
+The PXE boot installation works on unprepared machines. However, it can only be performed in certain network configurations:
```

Comment:
I must really speak a terrible Englis, because there are several places like this where my prose leads to a complete misunderstanding.

I meant "it works on unformatted machines"